test.sh

```diff
@@ -3,6 +3,10 @@
 echo "*** Running custom_active_record_observer specs"
 
 bundle install                                      || exit 1
+bundle exec appraisal install                       || exit 1
```

Comment:
Maybe, remove `bundle install` and `bundle exec appraisal install` from here and add them to `cache` options in `travis.yml`?